src/mode.js

```diff
@@ -51,18 +51,19 @@ export function setModeForTesting(m) {
  * @return {!ModeDef}
  */
 function getMode_() {
-  const isLocalDev = (location.hostname == 'localhost' ||
-      (location.ancestorOrigins && location.ancestorOrigins[0] &&
-          location.ancestorOrigins[0].indexOf('http://localhost:') == 0)) &&
+  const hostname = location.hostname;
```

Comment:
Will be reverted, this change was to allow flexible local development, the codebase has assumptions that host "**ads.localhost**" will always be used for development which makes staging/testing the stack cumbersome.

Are there gulp environment flags available to enforce a local development environment?